static/js/controllers/appcontroller.js

```diff
@@ -58,6 +58,7 @@ define(["jquery", "angular", "underscore"], function($, angular, _) {
 				}
 			}
 		};
+		$scope.defaults.settings.sounds = playSound.getDefaults();
```

Comment:
Returning now an object literal instead of a reference for `getDefaults`. This is where settings are initialized. I moved the declaration above into the `$scope.defaults`.